components/eamxx/src/share/io/scorpio_input.cpp

```diff
@@ -503,7 +503,47 @@ void AtmosphereInput::set_decompositions()
   for (int idof=0; idof<local_dim; ++idof) {
     offsets[idof] = gids_h[idof] - min_gid;
   }
-  scorpio::set_dim_decomp(m_filename,decomp_dim,offsets);
+
```

Comment:
I don't think this is necessary anymore. After #7175, we should be able to safely decompose any dimension.